docs/source/features/long_context_recipe.rst

```diff
@@ -0,0 +1,108 @@
+.. _long_context_recipe:
+
+Long Context Recipe
+------------------------
+
+Long context model training enhances the capability of large language models to handle long context inputs. Longer sequence lengths are beneficial for many NLP tasks, such as document-level summarization, long document classification, and long document question answering. NeMo Framework provides a recipe to train long context models like Llama-3, Mixtral, and Nemotron.
+
+Access Long Context Recipe
+========================
+
+NeMo 2.0 is providing tested recipes to train long context models. `The recipe is available in the NeMo LLM GPT Trainer Directory <https://github.com/NVIDIA/NeMo/tree/main/nemo/collections/llm/recipes>`_ directory.
```

Comment:
remove "Directory" from link. it is not needed.

NeMo 2.0 is providing tested recipes to train long context models. `The recipe is available in the NeMo LLM GPT Trainer <https://github.com/NVIDIA/NeMo/tree/main/nemo/collections/llm/recipes>`_ directory.